template/php/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM php:7-alpine
+
+# Composer requirements
+ARG COMPOSER_AUTH='{}'
+ENV COMPOSER_AUTH=${COMPOSER_AUTH}
+RUN curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer
```

Comment:
Would it be worth using multi-stage builds to pull this in from the composer image?

https://medium.com/@othillo/adding-composer-to-php-docker-images-using-multi-stage-builds-2a10967ae6c1